Mathlib/Topology/CWComplex/Classical/Basic.lean

```diff
@@ -565,23 +747,25 @@ lemma RelCWComplex.skeletonLT_monotone [RelCWComplex C D] : Monotone (skeletonLT
   fun _ _ h ↦ skeletonLT_mono h
 
 lemma RelCWComplex.skeleton_mono [RelCWComplex C D] {n m : ℕ∞} (h : m ≤ n) :
-    skeleton C m ⊆ skeleton C n :=
+    (skeleton C m : Set X) ⊆ skeleton C n :=
   skeletonLT_mono (add_le_add_right h 1)
 
 lemma RelCWComplex.skeleton_monotone [RelCWComplex C D] : Monotone (skeleton C) :=
   fun _ _ h ↦ skeleton_mono h
 
-lemma RelCWComplex.skeletonLT_subset_complex [RelCWComplex C D] {n : ℕ∞} : skeletonLT C n ⊆ C := by
-  simp_rw [← skeletonLT_top (C := C) (D := D)]
+lemma RelCWComplex.skeletonLT_subset_complex [RelCWComplex C D] {n : ℕ∞} :
+    (skeletonLT C n : Set X) ⊆ C := by
+  simp_rw [← skeletonLT_top]
```

Comment:
This feels like something which should be a general lemma on subcomplexes. I'm not saying this specific one should be deleted, but the general lemma should exist.